server/test/socialauth.spec.js

```diff
@@ -0,0 +1,48 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+
+chai.use(chaiHttp);
+const { expect } = chai;
+describe('POST /api/users/login/auth/response', () => {
+  it('it should return a user', (done) => {
+    chai.request(app)
+      .post('/api/auth/response')
+      .send({
+        email: 'cotutu50@gmail.com',
+        firstName: 'dinma',
+        lastName: 'jane',
+        username: 'dinma jane',
+      })
+      .end((err, res) => {
+        const newUser = res.body.user;
+        expect(res.status).to.equal(200);
+        expect(newUser).to.be.an('object');
+        expect(newUser.email).to.equal('cotutu50@gmail.com');
+        expect(newUser.firstName).to.equal('dinma');
+        expect(newUser.lastName).to.equal('jane');
+        expect(newUser.username).to.equal('dinma jane');
+        expect(newUser).to.have.property('token');
+        done();
+      });
+  });
+});
+
+describe('POST /api/users/login/auth/response', () => {
+  it('it should return a user', (done) => {
+    chai.request(app)
+      .post('/api/auth/response')
+      .send({
```

Comment:
Assign 
```
{
        email: 'cotutu50@gmail.com',
        firstName: 'dinma',
        lastName: 'jane',
        username: 'dinma jane',
      }
```
to variable since its used more than once